corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -159,6 +165,7 @@ <h2>{% trans "Manage Organization Structure" %}</h2>
 
     <div class="col-xs-7 col-md-7 col-lg-8">
 
+        <div>
```

Comment:
What does this wrapping `<div>` give us? I think we try not to include intermediate divs that don't use the bootstrap `col-*-*` sizing declarations, but that's not a hard and fast rule.

If this `<div>` _is_ necessary for the layout and you decide not to remove it, I'd suggest indenting the `<select>` as well to be a the right logical level.